install/ubuntu-installation-guide.md

```diff
@@ -0,0 +1,53 @@
+# Installing Kata Containers on Ubuntu
+
+Note:
+Kata Containers is available for Ubuntu\* **16.04** and **17.10**.
```

Comment:
Again, we should try to get packages for 18.04 LTS now that's been released.

Reply:
@jodh-intel changes applied, about the 18.04 LTS packages is something that we do not have currently maybe is a good idea to open an issue for @jcvenegas

Reply:
@GabyCT, @jcvenegas - https://github.com/kata-containers/packaging/issues/29 raised to track that.